meteoritics-and-planetary-science.csl

```diff
@@ -125,8 +129,8 @@
   </citation>
   <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="1">
     <sort>
-      <key macro="author"/>
-      <key variable="title"/>
+      <key macro="author-short" names-use-first="1" names-min="2"/>
+      <key macro="year-date"/>
```

Comment:
MaPS in-line citations are chronological, so it _should_ sort as (Zang 2000; Kita 2005). 
I thought fixing the disambiguation in the in-line citation would help the biblio sorting. There are still issues. Ideally, it would sort by first author (disambiguating by first name is authors have same surnames), then sorting within the authorship by date only (i.e., not considering latter authors).
For the biblio, using `<key macro="author"/>` is the better way, but does not work correctly in the following example:

> Zhang et al. 2012, 2016; H. Zhang et al. 2015; J. Zhang et al. 2015

is cited as (removing all but the 1st and 2nd authors and date):

> Zhang H., Yang Y., ... 2015. ...
Zhang J., Dauphas N., ... 2012. ...
Zhang J., Wang T., ... 2016. ...
Zhang J., Yang W., ...2015. ...

Note the 2015 reference is cited after the 2016 reference because of the 2nd author. Similar behavior occurs if using the "author-short" macro to sort.